firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -0,0 +1,443 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+// Selectors
+
+// Widget buttons
+let editWidgetButton = "com.apple.springboardhome.application-shortcut-item.configure-widget"
+let editHomeScreenButton = "com.apple.springboardhome.application-shortcut-item.rearrange-icons"
+let removeWidgetButton = "com.apple.springboardhome.application-shortcut-item.remove-widget"
+
+// Buttons Identifier
+let goToCopiedLink = "Go to Copied Link"
+let privateModeButtonToggle = "FirefoxHomepage.OtherButtons.PrivateModeToggle"
+let privateModeButtonToggleiPad = "TopTabsViewController.privateModeButton"
+
+// Widget coordinates
+let normalized = springboard.coordinate(withNormalizedOffset: CGVector(dx: 0, dy: 0))
+
+// Get the screen size
+let screenSize = springboard.windows.element(boundBy: 0).frame.size
+
+// Calculate the center-right coordinate (x: right edge, y: middle of the screen)
+let centerRightX = screenSize.width * 0.95  // Adjust this value if you want slightly away from the edge
+let centerRightY = screenSize.height / 2
+
+// Create the coordinate using the calculated points
```

Comment:
I guess this is device specific.. we should add which device we are testing currently in the comment

Reply:
IT is working on iPhone 15 and on the iPad. As you can see the code get the screen size before calculating the coordinates